images-list

```diff
@@ -52,48 +63,70 @@ istio/pilot rancher/istio-pilot 1.7.3
 istio/pilot rancher/istio-pilot 1.7.6
 istio/pilot rancher/istio-pilot 1.8.1
 istio/pilot rancher/istio-pilot 1.8.3
+istio/pilot rancher/mirrored-istio-pilot 1.8.3
 istio/proxyv2 rancher/istio-proxyv2 1.7.0
 istio/proxyv2 rancher/istio-proxyv2 1.7.1
 istio/proxyv2 rancher/istio-proxyv2 1.7.3
 istio/proxyv2 rancher/istio-proxyv2 1.7.6
 istio/proxyv2 rancher/istio-proxyv2 1.8.1
 istio/proxyv2 rancher/istio-proxyv2 1.8.3
+istio/proxyv2 rancher/mirrored-istio-proxyv2 1.8.3
 jaegertracing/all-in-one rancher/jaegertracing-all-in-one 1.20.0
+jaegertracing/all-in-one rancher/mirrored-jaegertracing-all-in-one 1.20.0
 jettech/kube-webhook-certgen rancher/jettech-kube-webhook-certgen v1.2.1
 jettech/kube-webhook-certgen rancher/jettech-kube-webhook-certgen v1.5.0
+jettech/kube-webhook-certgen rancher/mirrored-jettech-kube-webhook-certgen v1.2.1
 jimmidyson/configmap-reload rancher/jimmidyson-configmap-reload v0.3.0
 jimmidyson/configmap-reload rancher/jimmidyson-configmap-reload v0.4.0
+jimmidyson/configmap-reload rancher/mirrored-jimmidyson-configmap-reload v0.3.0
```

Comment:
We need both 0.4.0 and 0.3.0

Reply:
I went through the versions in rancher/charts and didn't seen v0.4.0 for this image. Do I need to do this for all versions, not just the ones in rancher/charts?

Reply:
https://github.com/rancher/charts/pull/1003/files#diff-5bb2300e45effc1f02a9d2fd7e3ca39d7efc30c030d641a30164056cc332f367R67
We should scan that other PR for all these images and see if they are used in multiple places

Reply:
Sorry. I did that, but I missed one. I went through it again and this is the only one I missed.